src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Name": "Project with specific transformation",
+  "SampleName": "ProjectWithSpecificTransformation",
+  "Description": "This sample demonstrates how to use the GeometryEngine with a specified geographic transformation to transform a geometry from one coordinate system to another. ",
+  "Instructions": "See https://developers.arcgis.com/net/latest/wpf/guide/coordinate-systems-and-transformations.htm for more information about geographic coordinate systems, geographic transformations, and projected coordinate systems. ",
```

Comment:
Is it possible to have these URLs formatted as links when output to the ReadMe.md?